TextEditorExtrasSettingsForm.inc.php

```diff
@@ -24,11 +24,31 @@ public function __construct($plugin) {
 	 * Settings are stored by context, so that each journal or press
 	 * can have different settings.
 	 */
-	public function initData() {
-		$contextId = Application::get()->getRequest()->getContext()->getId();
-		$this->setData('additions', $this->plugin->getSetting($contextId, 'additions'));
-		parent::initData();
-	}
+    public function initData()
```

Comment:
In `stable-3_3_0` we are still using tabs not spaces. Can you convert to tabs so we're consistent with other code in this file?